deselected_tests.yaml

```diff
@@ -71,21 +71,21 @@ deselected_tests:
   # SVR.fit fails when input is two samples of one class
   - preprocessing/tests/test_data.py::test_cv_pipeline_precomputed
 
-  # kdtree kNN rarely misses 0-distance points when kneighbors is used on same fitting data
+  # KDtree kNN rarely misses 0-distance points when kneighbors is used on same-fitting data
   - manifold/tests/test_spectral_embedding.py::test_precomputed_nearest_neighbors_filtering
 
-  # cache directory is not accessible on some systems
+  # Cache directory is not accessible on some systems
   - utils/tests/test_validation.py::test_check_memory
 
-  # oneDAL doesn't throw error if resulting coeffs are not finite
+  # OneDAL doesn't throw error if resulting coeffs are not finite
```

Comment:
`oneDAL/sklearn/sklearnex` are short names of libraries and should start from lower case even in the beginning of sentence.

Reply:
> 1. Generally, logically it is better separate into different PRs refactoring work and enhancement/bug fixing even in the same file.
> 2. If we tests is non-relevant now, we should also cover functionality with the new one as well.

I've created an Epic just for this purpose. A new ticket has been included to address this issue in detail, among 11 others within the file.

Reply:
Changed.